### ISSUE TITLE
Russian localization

### ISSUE THREAD

**Issue by devastaza** [radimitrov/CSharpShellApp#307]
Hi! Can i help you to fix some localization problems in app? I remember in old versions you had xml (or json) localization files. If you want, you can send me it.

**Comment by radimitrov** [comment]
Yes, it is still JSON. However it would be best wait after the next major update since there will be a lot of new fields from it and other recent updates.

**Comment by devastaza** [comment]
Sure, i'll wait

**Comment by devastaza** [comment]
> Yes, it is still JSON. However it would be best wait after the next major update since there will be a lot of new fields from it and other recent updates.

So i assume the time came in. I already found .json file and will localize it in my free time.

**Comment by devastaza** [comment]
> Yes, it is still JSON. However it would be best wait after the next major update since there will be a lot of new fields from it and other recent updates.

So i assume the time came in. I already found .json file and will localize it in my free time.

**Comment by devastaza** [comment]
> I haven't had the time to make sure there are no mistakes and regenerate the translations, which is slow. Plus, there have been new additions constantly. Right now some of the entries don't exist in the translations will fall back to the English JSON

Periodically i will check file for new lines.
